server/models/migrations/next/migrate.sql

```diff
@@ -16,4 +16,14 @@ INSERT INTO unit VALUES
  * @author: mbayopanda
  * @date: 2020-06-11
  */
-DELETE FROM unit WHERE id = 162;
\ No newline at end of file
+DELETE FROM unit WHERE id = 162;
```

Comment:
I had to make some changes to get this migrate file to run on our production databases.  Could you just add them here?

First, put 
```sql
DELETE FROM role_unit WHERE unit_id = 162;
```
right before this line.  It fails if users have a role assigned to the old stock management.